lib/kubot.rb

```diff
@@ -1,11 +1,18 @@
-
+require 'sinatra'
 require 'clamp'
 require 'slack-ruby-bot'
-
 require 'kubot/version'
-require 'kubot/bot'
 require 'kubot/main'
+require 'kubot/server'
+require 'eventmachine'
+require 'kubot/auth'
```

Comment:
it's always group requires like by prefix "kubot/" and have another group for third_party dependencies.